entity-types/ext-dbt_failed_resource_run/definition.yml

```diff
@@ -0,0 +1,67 @@
+domain: EXT
+type: DBT_FAILED_RESOURCE_RUN
+
+synthesis:
+  rules:
+    - identifier: entity_id 
+      name: entity_name 
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: dbt_resource_run 
+        - attribute: entity_name
+          present: True
```

Comment:
Oh. Incoming data has entity_name and entity_id in the source. To make synthesis more specific, I added this condition in case someone else want to create a custom event with the same name that shouldn't apply to this entity. I can remove it, but I wonder if having entity_id and entity_name in the source data could cause a problem.

Reply:
Just reread your comment and I understand now. I'll remove it.